src/client/src/commons/form/profile/components/profileLayers/components/chronoStratigraphyLayers/lithologyLayers.js

```diff
@@ -0,0 +1,236 @@
+import React, { useEffect, useRef } from "react";
+import * as Styled from "./lithologyLayersstyles";
+import PropTypes from "prop-types";
+import Draggable from "react-draggable";
+import { NumericFormat } from "react-number-format";
+import { useLayers } from "../../../../../../../api/fetchApiV2";
+import { CircularProgress } from "@mui/material";
+
+const LithologyLayers = props => {
+  const { stratigraphyId, navigationState, setNavigationState, unit } = props;
+
+  const { data } = useLayers(stratigraphyId);
+
+  const element = useRef(null);
```

Comment:
Wenn ich dich richtig verstanden habe, hast du hier viel Code von der ursprünglichen Stratigraphy Komponente übernommen. Vielleicht machen wir noch ein Issue ins Backlog damit man die wiederverwenden kann? Bzw irgendwie zusammenführen?

Reply:
Ja, ich hatte ursprünglich vor die Componente wiederzuverwenden, habe dann aber zu viele änderungen vornehmen müssen, sodass ich sie schlussendlich einfach kopiert habe. Ich denke die ursprüngliche Stratigraphy Komponente wird in Zukunft von dieser hier ersetzt. Ich denke nicht, dass ein zusammenführen sinn macht.